axelrod/strategies/axelrod_first.py

```diff
@@ -522,15 +523,15 @@ def __init__(self, alpha: float=0.05) -> None:
         """
         Parameters
         ----------
-        alpha, float
-            The significant level of pvalue from chi-squared test
-            0.05 by default according to literature
+        alpha: float
+            The significant level of p-value from chi-squared test
+            005 by default according to literature
         """
         super().__init__()
         self.alpha = alpha
-        self.opponent_is_random = None
```

Comment:
This is tripping the test. I suggest changing the test to `assertFalse`. (You did this on line 389 but not on line 385).